src/pages/Pages.tsx

```diff
@@ -28,7 +28,7 @@ function Pages() {
             <Route path="/login" element={<Login />} />
             <Route path="/quiz" element={<Quiz />} />
             <Route path="/shop" element={<Shop />} />
-            <Route path="/supplement" element={<Supplement />} />
+            <Route path="/:slug" element={<Supplement />} />
```

Comment:
could this be `/supplement/:slug` so it's not matching a wide set of paths? right now, routes like `/404` or `/anything-else` will match here

Reply:
Speaking of, it would be nice to add a catch-all route and include a general page not found screen `<Route path="*" element={<NotFound />} />`